Adyen/Analytics/AnalyticsProvider/AnalyticsProvider.swift

```diff
@@ -27,8 +27,10 @@ public struct AnalyticsConfiguration {
 @_spi(AdyenInternal)
 public protocol AnalyticsProviderProtocol: TelemetryTrackerProtocol {
     
-    var checkoutAttemptId: String? { get }
+    /// The payment-amount to be tracked
+    var amount: Amount? { get set }
```

Comment:
do we need "set" ?

Reply:
https://github.com/Adyen/adyen-ios/pull/1287/files#diff-3a0d5cb797232c8b37393f98b8fa3b114fb17f74fa72d82b016231ee5bf39448R64

This line nees it as the `AdyenContext` receives `any AnalyticsProviderProtocol`

Reply:
fair, it should not be possible to change amount on Context, but for now it is the only way 🤔